### ISSUE TITLE
CiteDCAT-AP Governance information

### ISSUE THREAD

**Issue by jakubklimek** [ec-jrc/datacite-to-dcat-ap#6]
Is there any kind of commitment of JRC to evolve CiteDCAT-AP as both DataCite and DCAT-AP evolve? Future milestones, etc?

The aim is to get and idea of what to expect, if one was to commit to using CiteDCAT-AP as a base data model for a future EOSC-compliant research data repository.

**Comment by H-a-g-L** [comment]
Yes JRC is committed to maintaining the mapping. Evaluation of alignment with DCAT-AP 3.0 (particularly DatasetSeries) is ongoing. A revision is also planned for when DataCite kernel 4.5 will be released. Thanking @andrea-perego for support

**Comment by jakubklimek** [comment]
Thanks, that is good to hear. Shouldn't this commitment be explicit in the specification?

**Comment by hanavyc** [comment]
Hi, I have two other questions about the same topic. Is it planned to release the official version of DataCite to DCAT-AP Mapping and the DataCite profile of DCAT-AP (CiteDCAT-AP)? And does JRC plan to cooperate on this work with DataCite?
